release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4066,3 +4066,31 @@ $ oc adm release info 4.12.33 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-34"]
+=== RHBA-2023:5151 - {product-title} 4.12.34 bug fix update
+
+Issued: 2023-09-20
+
+{product-title} release 4.12.34 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5151[RHBA-2023:5151] advisory. There are no RPM packages for this release.
```

Comment:
Obligatory note that this link 404s, but I know that it should be available on release day